### ISSUE TITLE
[Epic] Contract Explorer Invoke Contract 

### ISSUE THREAD

**Issue by janewang** [stellar/laboratory#1195]
### What problem does your feature solve?

Create Contract Explorer > Invoke Contract 

<img width="603" alt="Image" src="https://github.com/user-attachments/assets/a2e51c17-eed9-449b-8024-fb05b92a8b65" />

**Comment by jeesunikim** [comment]
new design: https://www.figma.com/design/yuikaFOPmJifyAfS90ER6T/Laboratory

**Comment by jeesunikim** [comment]
Are the buttons supposed to take users to `Transactions page` to sign? Where are they supposed to input their public/secret key?

**Comment by sdfcharles** [comment]
@jeesunikim yes i believe it would take the user to `Transactions` to sign cc @janewang

**Comment by jeesunikim** [comment]
We discussed this in today's meeting. We're going to ask people to connect wallet!
